framework/src/wifi_manager/wifi_manager.c

```diff
@@ -378,6 +382,11 @@ wifi_manager_result_e wifi_manager_disconnect_ap(void)
 
 	wifi_mutex_acquire(w_mutex, WIFI_UTILS_FOREVER);
 	result = wifi_utils_disconnect_ap();
+#ifdef CONFIG_ENABLE_IOTIVITY
+	if (result == WIFI_UTILS_SUCCESS) {
```

Comment:
I wonder this msg sending logic needs to be protected by w_mutex.

Reply:
no it doesn't need to be inside. I will fix